packages/components/src/components/DiagramSequence.vue

```diff
@@ -97,6 +97,12 @@ export default {
     },
   },
 
+  data() {
+    return {
+      collapsedActionState: [],
```

Comment:
For my edification - how does this component know to redraw when the set of collapsed actions changes?

Reply:
Though I'm not an expert in Vue, this is my understanding. 

For `collapsedActionState` to be reactive, it must be present in data function's return object. Since it's an array, settting elements like `this.collapsedActionState[5] = true`, without recreating the array reference, is not detected by Vue. Changes are detected - and a new render request is triggered - when we use `this.set$(collapsedActionState, 5, true)` instead.